packages/api/bin/cli.js

```diff
@@ -83,16 +83,21 @@ program
 program
```

Comment:
In reviewing previous migrations, it became apparent that the current approach might be problematic for users that wanted to run migration 4, but not 1 or 2.    We could certainly be defensive in programming migrations, but at some point the approach needs fortifying.      There will be a ticket out of this story to improve the migrations, however for now updating the upgrade docs and allowing users to run manually seemed a sane approach.

Reply:
👍